### DISCUSSION TITLE
Running vercel dev does not respect dev script in package.json

### DISCUSSION THREAD

**Discussion by mcabrams** [vercel/vercel#5302] [Help] [answered]
Running `vercel dev` does not seem to use the `dev` script I have specified in my package.json file like so:

```
  "scripts": {
    "dev": "concurrently \"npm run generate-dev\" \"next dev\"",
    ....
  }
```

Instead, I always see the default dev command being used:
```
➤ vercel dev                                                                                          13:48:46
Vercel CLI 20.1.2 dev (beta) — https://vercel.com/feedback
> NOTE: Requested port 3000 is already in use
> Running Dev Command “next dev --port $PORT”
```

Am I misunderstanding something?  Thanks!

**Comment by styfle** [comment, marked as answer]
Hi @mcabrams 

This is the expected behavior.

You can configure the Development Command in your project settings on the dashboard https://vercel.com/docs/build-step#build-&-development-settings

**Reply by mcabrams** [reply]
@styfle Gotcha, here is the documentation I saw that led me to believe otherwise, FWIW:

![image](https://user-images.githubusercontent.com/1213498/96771788-07654980-1397-11eb-8f19-0a7f4ab7213f.png)
Source: (https://vercel.com/docs/serverless-functions/introduction)

**Reply by styfle** [reply]
I see what you mean, that is confusing! We'll make sure to update the docs, thanks!
